components/collapsible/collapsible.test.js

```diff
@@ -144,6 +153,16 @@ describe('Dialtone vue Collapsible Component Tests', function () {
     });
   });
 
+  describe('If anchor text and anchor slot content are falsy', function () {
+    it('should output error message', async function () {
+      const consoleErrorSpy = sinon.spy(console, 'error');
+      props.anchorText = undefined;
+      _mountWrapper();
```

Comment:
Should be doing setup stuff such as this within a beforeEach in the describe. Ideally the `it()` should just be an assertion and that's it.